requirements.txt

```diff
@@ -6,3 +6,5 @@ youtube-dl>=2019.4.17
 jupyterlab
 opencv-python>=3.3.0.10
 pillow
+flask
```

Comment:
This should not be here, requirements.txt is only for Jupyter related dependencies.